docs/troubleshooting.md

```diff
@@ -62,3 +63,79 @@ For instance, try this:
 # oc create --raw /apis/admission.hive.openshift.io/v1/dnszones -f config/samples/hiveadmission-review-failure.json -v 8 | jq
 ```
 
+## Regaining access to a cluster after changing the API certificate
+
+There are several ways to regain access to a cluster after its API certificate has been changed.  Usually this will manifest as an error message similar to: `x509: certificate signed by unknown authority`.
+
+You will need your new certificate in standard pem format.  It should look something like this:
+
+```sh
+$ cat new-certificate.pem
+```
+
+```
+-----BEGIN CERTIFICATE-----
+<base64 encoded certificate blob>
+-----END CERTIFICATE-----
+...
+-----BEGIN CERTIFICATE-----
+<potentially more base64 encoded certificate blobs>
+-----END CERTIFICATE-----
+```
+
+### Replacing the API certificate of a managed single cluster with kubeconfig
+
+In order for a new certificate for a cluster to be accepted by the system, it must be added to the kubeconfig, alongside or replacing any existing certificates. Start by inspecting your current kubeconfig:
```

Comment:
We should distinguish that we need the CA certificate to be added to the kubeconfig.

```suggestion
In order for a new API certificate to be accepted by the system, the CA certificate used to sign the new API certificate must be added to the kubeconfig, alongside or replacing any existing certificates. Start by inspecting your current kubeconfig:
```